packages/terra-date-input/src/DateInput.jsx

```diff
@@ -696,6 +702,7 @@ class DateInput extends React.Component {
         showIsInvalid
         isIncomplete={this.props.isIncomplete}
         required={this.props.required}
+        ariadescribedby={this.props.customProps['aria-describedby']}
```

Comment:
nit : can we get this extract on top instead of accessing it through object

Reply:
Updated.